src/test/kotlin/cacophony/regex/RegexStringsTest.kt

```diff
@@ -0,0 +1,26 @@
+package cacophony.regex
+
+import cacophony.token.TokenCategoryGeneral
+import cacophony.token.TokenCategorySpecific
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+internal class RegexStringsTest {
+    @Test
```

Comment:
And I think we agreed to name test functions with names like: `test special character regex`, maybe without word 'test' (you can check in other branches)